ui/static/ui/js/utils.jsx

```diff
@@ -146,10 +147,16 @@ define("utils", ["jquery", "lodash", "react", "react_infinite", "select2"],
     },
     render: function() {
       var title = '';
+      var description = '';
       if (this.props.title) {
         title = <h4 className="modal-title">{this.props.title}</h4>;
       }
 
+      if (this.props.description) {
+       description = <p className="confirmation-description">
+         {this.props.description}
+       </p>;
+      }
```

Comment:
failing tests due to indentation here: https://travis-ci.org/mitodl/lore/builds/80808186#L1059